devops/ansible.yml

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
I don't love the name of this - could we do something like `deploy-playbook.yml` instead? I missed this on RRE so maybe we could update it there too?